dao/seeds/source_types.yml

```diff
@@ -302,13 +302,28 @@ oracle-cloud-infrastructure:
             initialValue: ocid
             name: authentication.authtype
           - component: text-field
-            isRequired: true
-            label: Oracle Cloud ID (OCID)
+            hideField: true
+            initializeOnMount: true
+            initialValue: ocid.uuid
             name: authentication.username
+          - component: text-field
+            name: application.extra.bucket
+            label: Bucket Namespace
+            isRequired: true
+            validate:
+              - type: required
+          - component: text-field
+            name: application.extra.bucket_namespace
+            label: Bucket Name
+            isRequired: true
+            validate:
+              - type: required
+          - component: text-field
+            name: application.extra.bucket_region
+            label: Bucket Name
+            isRequired: true
             validate:
               - type: required
-              - type: pattern
-                pattern: "^ocid1\\..*"
```

Comment:
Did we validate that the ID isn't like this pattern? It's ok if it isn't - just want to make sure.

Reply:
sweeeet sounds good!